### PR TITLE
TreeView of Debug Watch Style Defect problem

### LINKED ISSUE #167724: TreeView of Debug Watch Style Defect problem
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version:  1.74.0  ba9399b83d
- OS Version:  Mac 12.2.1 intel

Steps to Reproduce:

1. Open the debug activitybar of the local vscode. If the currently opened project does not have `.vscode/launch.json`, you need to create a new one to activate debug.
2. Add a test Expression, and watch carefully.
3. Found problems:

1. The end of the text is truncated because of italics.
2. The icon container is not vertically centered.

image

### PR DESCRIPTION
**Pull request by yiliang114**
Fixed #167724

### CODE REVIEW

**Comment by yiliang114** [comment]
After modification: 
image

**Comment by roblourens** [comment]
Thanks for the PR! We are working on our November release at the moment, I will look at this next week or when I have the time.

File: src/vs/workbench/contrib/debug/browser/media/debug.contribution.css [hunk 1]
```diff
@@ -87,6 +87,8 @@
 .monaco-workbench.mac .debug-pane .monaco-list-row .expression,
 .monaco-workbench.mac .debug-hover-widget .monaco-list-row .expression {
 	font-size: 11px;
+	display: flex;
+	align-items: center;
```
Comment (roblourens) [inline src/vs/workbench/contrib/debug/browser/media/debug.contribution.css:91]: `.expression` is already flex, and the `align-items` breaks a lot of other expressions

image

File: src/vs/workbench/contrib/debug/browser/media/debugViewlet.css [hunk 2]
```diff
@@ -229,7 +229,6 @@
 }
 
 .debug-pane .monaco-list-row .expression .value {
-	overflow: hidden;
```
Comment (roblourens) [inline src/vs/workbench/contrib/debug/browser/media/debugViewlet.css:232]: This is needed so we get an ellipsis `...` on overflowing text

File: src/vs/workbench/contrib/debug/browser/media/debug.contribution.css (hunk 1 above)
Comment (yiliang114) [reply src/vs/workbench/contrib/debug/browser/media/debug.contribution.css:91]: Yes, it seems that there is such a problem...

**Comment by yiliang114** [comment]
There seems to be a problem with my previous method. I'll close it first.

**Pull request closed without merging**